src/Nri/Ui/Icon/V3.elm

```diff
@@ -412,15 +412,15 @@ arrowRight assets =
 
 
 {-| -}
-assignmentStartButtonPrimary : { r | assignmentStartButtonPrimary_svg : String } -> IconType
+assignmentStartButtonPrimary : { r | assignmentStartButtonPrimary_svg : Asset } -> IconType
```

Comment:
what was the problem? I think to remember that I had problems with icons that weren't clean. I had success by using https://jakearchibald.github.io/svgomg/. Maybe create an issue for that problem.

Edit: actually I don't know which script you ran. I had problems with the one in the monolith not here.